src/web-components/SelectImages/index.ts

```diff
@@ -0,0 +1,165 @@
+import { css, html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { SelectImagesStyles } from './styles';
+
+interface WalletHandle extends Asset {
+    active: any;
+    default: boolean;
+    image?: string;
+    imageUrl?: string;
+}
+
+interface Asset extends BasicAsset {
+    name: string;
+    count?: number;
+    utxo?: string;
+    image?: string;
+    src?: string;
+    price?: number;
+    cost?: number;
+    validUntilDate?: number;
+}
+
+interface BasicAsset {
+    policyId: string;
+    hex: string;
+}
```

Comment:
Let's actually move these to an /interfaces folder and not export them from the main index.ts file. We'll figure out how to pull in kora-labs-common later